content/v2/zones.markdown

```diff
@@ -117,3 +117,44 @@ Responds with HTTP 200, renders the zone file.
 <%= pretty_print_fixture("/getZoneFile/success.http") %>
 ~~~
 
+
+## Get zone distribution {#get-zone-distribution}
+
+    GET /:account/zones/:zone
+
+Checks if a zone is fully distributed across DNSimple nodes.
+
+### Parameters
+
+Name | Type | Description
+-----|------|------------
+`:account` | `integer` | The account id
+`:zone` | `string` | The zone name
+
+### Example
+
+Check the zone distribution for `example.com` in the account `1010`:
+
+    curl  -H 'Authorization: Bearer <token>' \
+          -H 'Accept: application/json' \
+          https://api.dnsimple.com/v2/1010/zones/example.com/distribution
+
+### Response
+
+Responds with HTTP 200 when the zone is fully distributed.
+
+~~~json
+<%= pretty_print_fixture("/getZoneDistribution/success.http") %>
+~~~
+
+Responds with HTTP 200 when the zone is not distributed.
+
+~~~json
+<%= pretty_print_fixture("/getZoneDistribution/failure.http") %>
+~~~
+
+Responds with HTTP 500 when the server failed to perform the check.
```

Comment:
This sounds scary (`500` is a number that always is).

Not sure how often we should expect this to happen but I'd clearly explain it or rephrase this to clearly state that it's an error or unexpected behaviour.

Reply:
We don't expect it very often. This kind of error is returned in case the API server fails to contact all the nodes of our DNS cluster.

Reply:
I agree with @jacegu. 500 is the generic error when something goes wrong. This is for example the error we return when our system crashed.

In this case, we need to return a different error that makes it clear that it's not the API crashing, but it's the remote system that did not respond.

502 seems to be the most appropriate error. Sadly it's also the error returned by Nginx when not being able to connect to our backend app, but I'm not sure there is a better error.